MonoGame.Framework/Utilities/Png/PngWriter.cs

```diff
@@ -206,5 +203,98 @@ private int CalculateBytesPerPixel()
                     return -1;
             }
         }
+
+        private void GetColorData(Texture2D texture2D)
```

Comment:
Note that we would want to lift this method out into a common location eventually.  So for example we could share it with the JPEG writer too.